pkg/plugins/executor/swagger.yml

```diff
@@ -1,4739 +1,4675 @@
+            lun:
+                description: lun represents iSCSI Target Lun number.
+                format: int32
+                type: integer
+            portals:
+                description: |-
+                    portals is the iSCSI Target Portal List. The portal is either an IP or ip_addr:port if the port
+                    is other than default (typically TCP ports 860 and 3260).
+                    +optional
+                items:
+                    type: string
+                type: array
+            readOnly:
+                description: |-
+                    readOnly here will force the ReadOnly setting in VolumeMounts.
+                    Defaults to false.
+                    +optional
+                type: boolean
+            secretRef:
+                $ref: '#/definitions/LocalObjectReference'
+            targetPortal:
+                description: |-
+                    targetPortal is iSCSI Target Portal. The Portal is either an IP or ip_addr:port if the port
+                    is other than default (typically TCP ports 860 and 3260).
+                type: string
+        title: Represents an ISCSI disk.
+        type: object
+    Inputs:
+        description: Inputs are the mechanism for passing parameters, artifacts, volumes from one template to another
+        properties:
+            artifacts:
+                $ref: '#/definitions/Artifacts'
+            parameters:
+                description: |-
+                    Parameters are a list of parameters passed as inputs
+                    +patchStrategy=merge
+                    +patchMergeKey=name
+                items:
+                    $ref: '#/definitions/Parameter'
+                type: array
+        type: object
+    IntOrString:
+        description: |-
+            +protobuf=true
+            +protobuf.options.(gogoproto.goproto_stringer)=false
+            +k8s:openapi-gen=true
+        properties:
+            IntVal:
+                format: int32
+                type: integer
+            StrVal:
+                type: string
+            Type:
+                $ref: '#/definitions/Type'
+        title: |-
+            IntOrString is a type that can hold an int32 or a string.  When used in
+            JSON or YAML marshalling and unmarshalling, it produces or consumes the
+            inner type.  This allows you to have, for example, a JSON field that can
+            accept a name or number.
+            TODO: Rename to Int32OrString
+        type: object
+    Item:
         description: |-
-          Container image name.
-          More info: https://kubernetes.io/docs/concepts/containers/images
-          This field is optional to allow higher level config management to default or override
-          container images in workload controllers like Deployments and StatefulSets.
-          +optional
+            +protobuf.options.(gogoproto.goproto_stringer)=false
+            +kubebuilder:validation:T
```

Comment:
These all seem to have gotten in-lined -- seemingly because `Time`, `CreationTimestamp` and `DeletionTimestamp` all actually have different descriptions (but didn't before for some reason)

Reply:
Annnd GH did a wonderful job of taking my in-line comment with whitespace ignored and moving it to a completely different place.... 😐